README.md

```diff
@@ -1,5 +1,29 @@
 # rbxlx-to-rojo
 Tool to convert existing Roblox games into Rojo projects by reading their `rbxlx` place files.
 
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need at least Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Before you can port your game into Rojo projects, you need a .rbxlx file. If you have an existing game that isn't an .rbxlx file, Go to studio, click on any place, and then click on File -> Save to file as.
```

Comment:
Please turn this into a list with bullet points, rather than the way it is now.

Reply:
Might have fixed this, check the latest commit.